tests/test_annotation_stores.py

```diff
@@ -1033,9 +1033,9 @@ def test_load_cases_error(fill_store, store_cls):
             store._load_cases(["foo"], lambda: None, lambda: None)
 
     @staticmethod
-    def test_py38_init(fill_store, store_cls, monkeypatch):
+    def test_py37_init(fill_store, store_cls, monkeypatch):
         """Test that __init__ is compatible with Python 3.8."""
-        py38_version = (3, 8, 0)
+        py37_version = (3, 7, 0)
```

Comment:
Interestingly, I think this should never have been 38 but should be fixed to 3.7/ Was it changed when deprecating 3.7 before? I think the aim of this test is to test for lack of the deterministic key word, which is missing before 3.8 (<= 3.7).

Reply:
Yes. I just manually reverted #344